CMakeLists.txt

```diff
@@ -41,7 +41,13 @@ set(CMAKE_C_STANDARD_REQUIRED ON)
 # may be useful to simplify the build and distribution of the application. E.g.
 # in test farms, or in cases where the end-user system configs are known in
 # advance. Use it with discretion.
-option (BUILD_SHARED_LIBS "Build shared libs" ON)
+include(CMakeDependentOption)
```

Comment:
Minor: I don't think this PR introduces any dependent options so perhaps this can be removed?

Reply:
Yeah this can go